examples/https/server.c

```diff
@@ -85,14 +131,18 @@ int https_server_test(int argc, char** argv)
     if (ret != 0) goto exit;
 
     /* setup listener */
-    ret = wolfSockListen(&listenFd, HTTPS_TEST_PORT);
+    ret = wolfSockListen(&listenFd, port);
     if (ret != 0) goto exit;
 
     do {
-        ret = etsi_client_get_all(etsiServer, etsi_key_cb, ctx);
-        if (ret != 0) {
-            mStop = 1;
-            goto end_sess;
+        if (useKeyMgr) {
+            ret = etsi_client_get_all(etsiServer, etsi_key_cb, ctx);
+            if (ret != 0) {
+                printf("\nFailure connecting to key manager\n");
```

Comment:
Since this is an error, I think it should go to stderr. (here and all other error output).